pkg/network/usm/kafka_monitor_test.go

```diff
@@ -359,6 +360,7 @@ func TestKafkaProtocolParsing(t *testing.T) {
 			},
 			teardown: kafkaTeardown,
 			configuration: func() *config.Config {
+				_, _ = sysconfig.New("/doesnotexist")
 				cfg := config.New()
```

Comment:
is that mandatory to have? if so, let's use `testConfig` everywhere

Reply:
Since the adjustments are happening in `cmd/system-probe/config` now, that logic needs to happen first. There is a good chance most of these tests don't care, but I did it just to be sure.

TBH, I don't love how easy it is to do the wrong thing, so I'm thinking about ways to improve this before I merge.

Reply:
My concern is that people won't be aware of that change (if that's required), and easily they can forget about that, and spend a lot of time just because they forgot to add this line

Reply:
Yeah, that is what I meant by "I don't love how easy it is to do the wrong thing".

Reply:
I made it harder to "use wrong", but I think long-term we will want the configs to be components. I started exploring that, but it is a much larger code change.